e2e/tests/lerna-clean/lerna-clean.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { Fixture } from "../../utils/fixture";
+import { normalizeEnvironment } from "../../utils/snapshot-serializer-utils";
+
+expect.addSnapshotSerializer({
+  serialize(str) {
+    return normalizeEnvironment(str.replaceAll(/index .{7}\.\..{7} \d{6}/g, "index XXXXXXX..XXXXXXX XXXXXX"));
+  },
+  test(val) {
+    return val != null && typeof val === "string";
+  },
+});
+
+describe("lerna-clean", () => {
+  let fixture: Fixture;
+
+  beforeAll(async () => {
+    fixture = await Fixture.create({
+      name: "lerna-clean",
+      packageManager: "npm",
+      initializeGit: true,
+      runLernaInit: true,
+      installDependencies: true,
+    });
+  });
+  afterAll(() => fixture.destroy());
+
+  it("should remove node_modules for all packages", async () => {
```

Comment:
Can we add an actual check for no more node_modules to these tests?

i.e. check they exist with contents before the clean command and don't exist afterwards